server/methods/followUser.js

```diff
@@ -0,0 +1,43 @@
+import { Meteor } from 'meteor/meteor';
+
+import { Users } from '../../app/models';
+import { settings } from '../../app/settings';
+
+
+Meteor.methods({
+	followUser(username) {
+		if (settings.get('Newsfeed_enabled')) {
```

Comment:
Please, never put all your code inside an if, use the if first to stop the process:
```
if (!settings.get('Newsfeed_enabled')) {
    return false;
}
```
It will make the code cleaner and more understandable.